src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -110,6 +110,7 @@ public override void Convert(Stream input, IResultLogWriter output, OptionallyEm
 
             _invocation = new Invocation();
             _invocation.ToolExecutionNotifications = new List<Notification>();
+            _invocation.ExecutionSuccessful = true;
```

Comment:
>_invocation.ExecutionSuccessful = true; [](start = 12, length = 39)

again, perhaps there is a smarter way to figure out the right status here, i simply didnt have the right knowledge about this to improve. Please cross check if we can have some logic here.